data/genres.json

```diff
@@ -3,14 +3,14 @@
     "id": 448,
     "name": "Alternative",
     "items_ids": [
-      506
+
     ]
   },
   {
     "id": 830,
     "name": "Pop",
     "items_ids": [
-      707
+
```

Comment:
Please make sure this information is in this file, and stays there when the program loads and closes